src/util/sortable_collectibles.ts

```diff
@@ -0,0 +1,72 @@
+import { BigNumber } from '0x.js';
+
+import { getCollectiblePrice } from './collectibles';
+import { Collectible } from './types';
+
+export interface SortableCollectible {
+    price: BigNumber | null;
+    creationDate: BigNumber | null;
+    collectible: Collectible;
+}
+
+export enum CollectibleSortType {
+    PriceLowToHigh = 'Price: low to high',
+    PriceHighToLow = 'Price: hight to low',
+    NewestAdded = 'Newest Added',
```

Comment:
I made the string values of the enum hold the text we are displaying in the UI. I'll probably change this later because it is generally not a very good idea. My suggestions is _do not add anything that depenes on the string value of this_.